### DISCUSSION TITLE
eToro CopyTrader and Smart Portfolios

### DISCUSSION THREAD

**Discussion by RealCommerzpunk** [ghostfolio/ghostfolio#3840] [Q&A]
Hello! I started using Ghostfolio just a few hours ago, and I like it a lot so far.

A part of my invenstment is made in eToro using their tools CopyTrader, which copies the investment of other people and Smart Portfolios which are curated Portfolios, similar to ETF.

Both of them have, as far as I can see, no ISIN. That seem to be logical.

But how can I add them to Ghostfolio and take track of the historical development?

Any Idea?

**Comment by dtslvr** [comment]
Would a custom asset be an option for this and manually update historical data?

See _How do I add a custom asset?_ in the [FAQ (Self-Hosting)](https://ghostfol.io/en/faq/self-hosting).

**Comment by RealCommerzpunk** [comment]
Okay, thank you, i will try this.
